### DISCUSSION TITLE
Sort branches based on most recent activity

### DISCUSSION THREAD

**Discussion by JosephTLyons** [community/community#13719] [Mobile]
I think it would be nice if the user could sort branches based on most recent activity.  In a repository with a lot of branches, it can be hard to find out where changes are happening at; being able to sort by activity would put those branches right at the top.

**Comment by brianlovin** [comment]
Great feedback, thanks @JosephTLyons — I've opened a tracking issue for our team to look into this and make this branches view better.

**Reply by JosephTLyons** [reply]
I suppose another very similar request would be to sort repositories, in the repository list, based on most recent activity.

**Reply by brianlovin** [reply]
100% — that list should also be searchable 😅

**Reply by JosephTLyons** [reply]
Yes! :). I think those few things would make mobile dramatically faster to use.
